### PR TITLE
Fix music_display not stretching properly, reconsolidate ForegroundLayer into InterjectionLayer, rename InterjectionLayer to SplashLayer

### PR DESCRIPTION
**Pull request by in1tiate**
i also wrote a big ass comment

### CODE REVIEW

**Review by oldmud0 — DISMISSED** [review]
Looks fine. (Please squash when merging)

**Comment by in1tiate** [comment]
investigating an issue causing chat_arrow to be incorrectly positioned

**Comment by in1tiate** [comment]
Also investigating an issue causing back-to-back shouts to not display (probably caused by continuous playback)

File: src/courtroom.cpp [hunk 1]
```diff
@@ -72,9 +72,13 @@ Courtroom::Courtroom(AOApplication *p_ao_app) : QMainWindow()
   ui_vp_testimony->setAttribute(Qt::WA_TransparentForMouseEvents);
   ui_vp_wtce = new SplashLayer(this, ao_app);
   ui_vp_wtce->set_play_once(true);
+  ui_vp_wtce->continuous = false;
+  ui_vp_wtce->force_continuous = true;
   ui_vp_wtce->setAttribute(Qt::WA_TransparentForMouseEvents);
   ui_vp_objection = new SplashLayer(this, ao_app);
   ui_vp_objection->set_play_once(true);
+  ui_vp_objection->continuous = false;
+  ui_vp_objection->force_continuous = true;
```
Comment (oldmud0) [inline src/courtroom.cpp:81]: Something seems fishy with this but I lack the braincells at the moment to understand my intuition

**Review by oldmud0 — APPROVED** [review]

File: src/courtroom.cpp (hunk 1 above)
Comment (in1tiate) [reply src/courtroom.cpp:81]: `continuous` is set automatically using a simple file path check (is this the same file we just loaded). if you want to bypass that check, you need to set `force_continuous` to true - if `force_continuous` is true, `start_playback()` will blindly use whatever the value of `continuous` already is and not perform any check at all

Comment (in1tiate) [reply src/courtroom.cpp:81]: the reason this smells fishy is because it is a terrible hack, but a necessary hack as far as I can tell

**Comment by stonedDiscord** [comment]
🚨 merge conflicts 🚨

**Review by Crystalwarrior — APPROVED** [review]
k merg conflicts resolved

**Pull request merged**